fecgov/fec-cms
# Update DEFAULT_TIME_PERIOD to be 2020

## Summary (required)

- Resolves #2640 
_Updating all instances of `DEFAULT_TIME_PERIOD` to be 2020 to show most up to date cycle information._

We have `DEFAULT_TIME_PERIOD` global variable referenced in both the main.jinja as a JS global var and one inside of constants.py as a python global var. Below I referenced which variable is being used on which impacted area of the application. All should now be defaulting to 2020.

In addition, I took the opportunity to do some clean up with macros and filters that were no longer being used and utilizing the global DEFAULT_TIME_PERIOD var for python files.

## Impacted areas of the application

Item | Link | Var reference
------ | ------ | ------
Raising line chart time period | http://localhost:8000/data/browse-data/?tab=raising | main.jinja
Spending line chart time period | http://localhost:8000/data/browse-data/?tab=spending | main.jinja
Transaction time period filter using partition_field macro | http://localhost:8000/data/disbursements/, http://localhost:8000/data/receipts/, http://localhost:8000/data/receipts/individual-contributions  | constants.py

## Screenshots

### Raising line chart
<img width="728" alt="Screen Shot 2019-05-24 at 11 59 57 AM" src="https://user-images.githubusercontent.com/12799132/58341233-c40e5080-7e1b-11e9-9ad9-ae163411aae6.png">

### Spending line chart
<img width="715" alt="Screen Shot 2019-05-24 at 12 00 11 PM" src="https://user-images.githubusercontent.com/12799132/58341246-c96b9b00-7e1b-11e9-9561-374a4eb8bc67.png">

### Transaction time period filter
<img width="222" alt="Screen Shot 2019-05-24 at 12 01 49 PM" src="https://user-images.githubusercontent.com/12799132/58341255-d12b3f80-7e1b-11e9-927e-600efa7d4b55.png">

## How to test
- Go to the pages that are referenced in the "Impacted areas of the application" section above in this PR.
- Verify that defaults are now going to 2020
